server/.env

```diff
@@ -2,4 +2,6 @@ TEAM_NAME="Aecio, Shums, Marco, Nicky, Derek, Ahana"
 
 MONGO_URL="mongodb://localhost:27017/mentions"
 
-SECRET="verysecretsecret"
\ No newline at end of file
+SECRET="verysecretsecret"
+
+SENDGRID_API_KEY="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"
```

Comment:
We probably shouldn't merge any api keys to our project. You might get emails from send grid saying the api key is exposed. Someone can also use the key for themselves and deplete the api usage quota.
Maybe we could share keys on slack. 
We should also gitignore the env file

Reply:
Alright, I added .env to .gitignore